components/TaskHeader.tsx

```
@@ -1,38 +1,26 @@
-          alt="header-icon"
-          width={20}
-          height={20}
-        />
-        <h2 className="text-[#4541C6] text-base sm:text-lg ml-2 font-medium">
-          {title}
-        </h2>
+        <Image src={icon || '/assets/ToDoEllipse.svg'} alt="header-icon" width={20} height={20} />
+        <h2 className="ml-2 text-base font-medium text-[#4541C6] sm:text-lg">{title}</h2>
       </div>
 
       <button onClick={onCreateTask} className="cursor-pointer">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)